task-launcher/src/tasks/shared/trials/afcStimulus.js

```diff
@@ -9,6 +9,8 @@ import { mediaAssets } from "../../..";
 import _toNumber from 'lodash/toNumber'
 import { camelize } from "@bdelab/roar-utils";
 import { getDevice } from "@bdelab/roar-utils";
+import { isMaxTimeoutReached } from '../helpers/appTimer';
```

Comment:
Remove this line, instead import in line 7 from helpers once exports are added to the index.js file.